debian/copyright

```diff
@@ -0,0 +1,7 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: pgautofailover-cli
+Source: https://github.com/citusdata/pg_auto_failover
+
+Files: *
+Copyright: 2012-2018, Citus Data, Inc.
```

Comment:
Copyright (c) Microsoft Corporation. All rights reserved.

Reply:
There is a warning by lintian on this one: https://lintian.debian.org/tags/copyright-without-copyright-notice.html
This could be solved by making it: `Copyright 2019 (c) Microsoft Corporation. All rights reserved.`

What should we do?